spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/batch/SimpleJobServiceFactoryBean.java

```diff
@@ -215,13 +216,10 @@ public void afterPropertiesSet() throws Exception {
 			lobHandler = new DefaultLobHandler();
 		}
 
-		// TODO: BOOT2 this may cause issues as it just removed for now
-		//if (serializer == null) {
-		//	XStreamExecutionContextStringSerializer defaultSerializer = new XStreamExecutionContextStringSerializer();
-		//	defaultSerializer.afterPropertiesSet();
-		//
-		//	serializer = defaultSerializer;
-		//}
+		if (serializer == null) {
```

Comment:
Can we have a test that verifies this, please?

Reply:
Added